package.json

```diff
@@ -5,9 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test:open": "npm run lint && cypress open",
+    "pretest": "rm -f cypress/reports/*",
     "test": "cypress run",
```

Comment:
Well, you need to use the scripts "pretest" and "posttest" you created to generate the reports

You need to create a pipeline in test and test:open scripts

Reply:
i did what you told me and the postest script throws me this 
![image](https://user-images.githubusercontent.com/46982083/182256750-14a8ce52-e9d7-4b32-ab51-9aa06f5c3bbb.png)